examples/test_gfc.cpp

```diff
@@ -0,0 +1,441 @@
+#include "mfem.hpp"
+#include <fstream>
+#include <iostream>
+
```

Comment:
I'm getting lots of output from
`mpirun -np 4 test_gfc -m ../data/fichera-q3.mesh`

Reply:
The test scheme assumes that the `GridFunction` can exactly represent the test function (which is linear).  With that mesh I think we need third order basis functions to handle the geometry.  If you add "-o 3" it should succeed.

Reply:
By the way, the main point of that test is to verify that our `GridFunctionCoefficient`s are returning the correct values when we evaluate them on boundary elements or interior face elements.  I wanted to make sure the `FaceElementTransformations` object was performing the transformation of the reference coordinates from the 2D face coordinates to the 3D neighboring element coordinates correctly and that the new `GridFunction::GetValues` method was using the `FaceElementTransformations` correctly.